### PR TITLE
Create locking between Lambda containers

### PR DESCRIPTION
**Pull request by AhyoungRyu**
@1ambda As you suggested I created a lock between Lambda containers. 

Let me briefly share the background history. 
As described in ["AWS Lambda Container Lifetime and Config Refresh"](https://www.linkedin.com/pulse/aws-lambda-container-lifetime-config-refresh-frederik-willaert), due to this reusing Lambda container mechanism, sometimes duplicated Helium pkgs are written again and again.  (as you sometimes noticed.. 😅)

With this patch, when [`helium.json`](https://s3.amazonaws.com/helium-package/helium.json)created below info will be saved under [`helium-metadata.json`](https://s3.amazonaws.com/helium-package/helium-metadata.json)
```
{
  "metaData": {
    "_createdAt": "2017-02-14T09:01:58+00:00",
    "_lambdaContainerId": "905149"
  }
}
```
Like you said in [this comment](https://github.com/apache/zeppelin/pull/1935#issuecomment-276862781), you can also check when `helium.json` was created with `_createdAt` value, 
The reason why I didn't directly added this `metaData` field to `helium.json` is to avoid changing Zeppelin source code. 

To sum up, if `_lambdaContainerId` (previous Lambda container id) and current one are same, then the running will be skipped. And will wait until the next trigger. 
screen shot 2017-02-14 at 6 30 40 pm

### CODE REVIEW

**Comment by 1ambda** [comment]
Read whole description, and looks nice.

**Comment by AhyoungRyu** [comment]
Thanks @1ambda!
